src/kOS/Screen/TermWindow.cs

```diff
@@ -133,6 +134,8 @@ public void Awake()
 
             GameEvents.onHideUI.Add (OnHideUI);
 			GameEvents.onShowUI.Add (OnShowUI);
```

Comment:
Some how this section previously used tabs instead of spaces, so the new portion that follows also uses tabs, I'll swap to spaces.